source/draw/Color.ooc

```diff
@@ -153,6 +153,33 @@ ColorRgba: cover {
 	distance: func (other: This) -> Float { (this toRgb() distance(other toRgb()) * 3.0f + (this a - other a) as Float pow(2)) / 4.0f sqrt() }
 	operator == (other: This) -> Bool { this equals(other) }
 	operator != (other: This) -> Bool { !this equals(other) }
+
+	fromText: static func (text: Text) -> This {
```

Comment:
I guess that you just moved this function but if you use Text, you have to free it when given ownership.